_layouts/activity.html

```diff
@@ -52,7 +52,7 @@
 				<h2>Downloads</h2>
 				<ul>
 					{% for doc in page.downloads %}
-					<li><a href="{{ site.url }}{{ site.baseurl }}/downloads/{{ doc.filename }}">{{ doc.name }}</a></li>
```

Comment:
This file should not have been edited (it is being reverted to a previous version here.)